homeassistant/bootstrap.py

```diff
@@ -197,10 +197,8 @@
 SETUP_ORDER = {
     # Load logging as soon as possible
     "logging": LOGGING_INTEGRATIONS,
-    # Setup frontend
-    "frontend": FRONTEND_INTEGRATIONS,
-    # Setup recorder
-    "recorder": RECORDER_INTEGRATIONS,
+    # Setup frontend and recorder
+    "frontend_and_recorder": {*FRONTEND_INTEGRATIONS, *RECORDER_INTEGRATIONS},
```

Comment:
This is kinda meh as we show it in the log.  This probably makes sense to be a flat list of dicts with a name and integrations